src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -98,6 +102,9 @@ export class ListOfComplaintsComponent implements OnInit {
             }
         }
         this.subscribeToComplaintResponse(complaintResponse);
+        this.courseManagementService.find(this.courseId).subscribe((response) => {
+            this.isAtLeastInstructor = response?.body?.isAtLeastInstructor ?? false;
+        });
```

Comment:
We should discuss if we need this call. While this is the only information to find out the user's access rights directly, we could use another option to avoid this call:

We expect the server to filter out information about the students and the reviewers already. We could use the existing flag `hasStudentInformation` to check if the server gave us this information, and only show them if we received them. This works, but we are also relying on the Server to do this filter correctly every time, and have no additional safety layer in the client. But this saves us one additional REST-call.

See https://github.com/ls1intum/Artemis/blob/edee8b4bbefbf5f6a3373fe47475c516139a7e1e/src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java#L459